cmd/ops_agent_windows/main_windows.go

```diff
@@ -64,17 +87,20 @@ func initServices() error {
 	}
 	// TODO: Write meaningful descriptions for these services
 	services = []struct {
-		name    string
-		exepath string
-		args    []string
+		name        string
+		displayName string
+		exepath     string
+		args        []string
 	}{
 		{
 			serviceName,
+			serviceDisplayName,
 			self,
 			[]string{"-in", filepath.Join(base, "../config/config.yaml"), "-out", configOutDir},
 		},
 		{
-			fmt.Sprintf("%s - Metrics Agent", serviceName),
+			fmt.Sprintf("%s-otel", serviceName),
```

Comment:
-metrics-agent? I don't think "-otel" is something users are going to understand.

Reply:
Sync'd offline. We'll go with `-opentelemetry-collector` here.

Reply:
Done.